src/.babelrc.json

```diff
@@ -27,14 +27,5 @@
       },
       "core"
     ],
-    [
```

Comment:
reviewers please confirm this is the correct removal

Reply:
This seems correct to me -- and since webpack isn't crashing it's definitely ok